js/ui/overlay/ui.overlay.js

```diff
@@ -1506,6 +1421,9 @@ const Overlay = Widget.inherit({
                 this._initContainer(value);
                 this._invalidate();
                 this._toggleSafariScrolling();
+                if(this.option('dragEnabled') && !this.option('dragArea')) {
+                    this._draggable?.updateDragContainer(this._getDragResizeContainer());
```

Comment:
what about `resizable` container update?

Reply:
fair enough, added container update for `resizable` also